doc(release) Release notes 24.10.0 Modules (#3720)
centreon/centreon-documentation: versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx

title: Commercial Extensions
---

## Introduction

You can find in this chapter all changelogs concerning **Centreon Commercial Extension**.

> It is very important when you update your system to refer to this section in order to learn about behavior changes or
> major changes that have been made on this version. This will let you know the impact of the installation of these
> versions on the features you use or the specific developments that you have built on your platform (modules,
> widgets, plugins).

If you have feature requests or want to report a bug, please contact support.

> Read more about version 24.10 in our [blog post](https://www.centreon.com/en/blog/centreon-fall24-whats-new-in-the-24-10-software-version/).

Reviewer: ```suggestion
Read more about version 24.10 in our [blog post](https://www.centreon.com/centreon-announces-its-latest-software-release-centreon-24-10/).
```

- [Server] Adapted Protobuf to handle the new event "AdaptiveServiceStatus" from Broker.
- [Server] Improved logs related to Broker event processing and made event queue size configurable.
- [Server] Improved the parsing of Broker events on creation and deletion of resources.
- [Server] Increased the default log level to INFO.
- [Server] Optimized the propagation of a resource status change.
- [Viewer] You can now display a host name for a service.
- [Viewer] You can now display the status as background color for a resource in weather style.

</details>

<details>
  <summary>Bug fixes</summary>

- [Server] Fixed a synchronization issue that could cause resources to disappear from a map.
- [Server] Fixed concurrent access issues causing exceptions in the viewer.
- [Server] Fixed Servicegroup status when configured using Service Templates or Hostgroup Services.

</details>

## Centreon BAM

### 24.10.0

Release date: `October 30, 2024`

<details open>
  <summary>Enhancements</summary>

- Business Activity Diagram new widget is now available in Dashboards.

</details>

## Centreon MBI

### 24.10.0

Release date: `October 30, 2024`

<details open>
  <summary>Enhancements</summary>

> - [MBI] Improved the starting of CBIS by using systemd configuration.

Reviewer: ```suggestion
- [MBI] The CBIS service is now started using systemd's configuration.
```

title: Commercial Extensions
---

## Introduction

You can find in this chapter all changelogs concerning **Centreon Commercial Extension**.

> It is very important when you update your system to refer to this section in order to learn about behavior changes or
> major changes that have been made on this version. This will let you know the impact of the installation of these
> versions on the features you use or the specific developments that you have built on your platform (modules,
> widgets, plugins).

If you have feature requests or want to report a bug, please contact support.

Read more about version 24.10 in our [blog post](https://www.centreon.com/centreon-announces-its-latest-software-release-centreon-24-10/).

<ExpandCollapseAll />

## Centreon MAP

### 24.10.0

Release date: `October 30, 2024`

<details open>
  <summary>Map Legacy retirement</summary>

- **MAP Legacy is no longer available.**
- If you are still using Map Legacy and want to know more about how to migrate to MAP, read [MAP Legacy end of life](https://docs.centreon.com/docs/graph-views/map-legacy-eol/).

> </details>

Reviewer: ```suggestion

</details>
```

## Introduction

You can find in this chapter all changelogs concerning **Centreon Commercial Extension**.

> It is very important when you update your system to refer to this section in order to learn about behavior changes or
> major changes that have been made on this version. This will let you know the impact of the installation of these
> versions on the features you use or the specific developments that you have built on your platform (modules,
> widgets, plugins).

If you have feature requests or want to report a bug, please contact support.

Read more about version 24.10 in our [blog post](https://www.centreon.com/centreon-announces-its-latest-software-release-centreon-24-10/).

<ExpandCollapseAll />

## Centreon MAP

### 24.10.0

Release date: `October 30, 2024`

<details open>
  <summary>Map Legacy retirement</summary>

> - **MAP Legacy is no longer available.**

Reviewer: ```suggestion
> **MAP Legacy is no longer available.**
```

### 24.10.0

Release date: `October 30, 2024`

<details open>
  <summary>Enhancements</summary>

- A new "Business Activity Diagram" widget is now available in Dashboards.

</details>

## Centreon MBI

### 24.10.0

Release date: `October 30, 2024`

<details open>
  <summary>Enhancements</summary>

- The CBIS service is now started using systemd's configuration.
- Packaged SELinux rules for centreon-map and centreon-mbi.

</details>

## Centreon Auto Discovery

### 24.10.0

Release date: `October 30, 2024`

<details open>
  <summary>Enhancements</summary>

- [API] Fixed access control for some v2 API endpoints.

</details>

## Centreon License Manager

### 24.10.0

Release date: `October 30, 2024`

<details>
  <summary>Compatibility</summary>

- Compatibility with other 24.10 components.

</details>

## Centreon Anomaly Detection

### 24.10.0

Release date: `October 30, 2024`

<details open>
  <summary>Enhancements</summary>

- Improved ergonomics by displaying the entire window with a default screen resolution, making all buttons visible.

</details>

## Centreon IT Edition Extensions

### 24.10.0

Release date: `October 30, 2024`

<details open>
  <summary>Enhancements</summary>

> - [Dashboards] A new PATCH endpoint has been added for updating playlists.

Reviewer: ```suggestion
**Dashboard playlists**
- Added unauthenticated playlists for TV projection.
- A new PATCH endpoint has been added for updating playlists.
```

- Improved logs related to Broker event processing and made event queue size configurable.
- Improved the parsing of Broker events on creation and deletion of resources.
- Increased the default log level to INFO for map-engine.
- Fixed access control for some v2 API endpoints.
- Packaged SELinux rules for centreon-map and centreon-mbi.

**Editor**
- You can now hide container labels.
- Increased the number of available fonts and made Roboto the default one.

**Viewer**
- You can now display a host name for a service.
- You can now display the status as background color for a resource in weather style.

**GeoViews**
- Redesigned the edit window.

</details>

<details>
  <summary>Bug fixes</summary>

- Fixed a synchronization issue that could cause resources to disappear from a map.
- Fixed concurrent access issues causing exceptions in the viewer.
- When using Service Templates or Hostgroup Services, the status of Service groups is now displayed correctly.

</details>

## Centreon BAM

### 24.10.0

Release date: `October 30, 2024`

<details open>
  <summary>Enhancements</summary>

**Dashboards**
- A new "Business Activity Diagram" widget is now available.

> - Added the handling of Business Activities and Business Views in the Status Grid widget.

Reviewer: ```suggestion
- Business Activities and Business Views can now be displayed in the Status Grid widget.
```
